pycbc/workflow/datafind.py

```diff
@@ -906,6 +906,8 @@ def run_datafind_instance(cp, outputDir, connection, observatory, frameType,
     # workflow format output file
     cache_file = File(ifo, 'DATAFIND', seg, extension='lcf',
                       directory=outputDir, tags=currTags)
+    cache_file.PFN(cache_file.cache_entry.path, site='local')
+    
```

Comment:
I also don't see why it would. Is there a reason the cache file couldn't be in place before running pycbc_submit_dax? If not this should have no adverse effects.